TTS/tts/utils/text/phonemizers/espeak_wrapper.py

```diff
@@ -163,6 +164,13 @@ def phonemize_espeak(self, text: str, separator: str = "|", tie=False) -> str:
 
             # dealing with the conditions descrived above
             ph_decoded = ph_decoded[:1].replace("_", "") + ph_decoded[1:]
+
+            # espeak-ng backend can add language flags that need to be removed:
+            #   "sɛʁtˈɛ̃ mˈo kɔm (en)fˈʊtbɔːl(fr) ʒenˈɛʁ de- flˈaɡ də- lˈɑ̃ɡ."
+            # phonemize needs to remove the language flags of the returned text:
+            #   "sɛʁtˈɛ̃ mˈo kɔm fˈʊtbɔːl ʒenˈɛʁ de- flˈaɡ də- lˈɑ̃ɡ."
+            ph_decoded = re.sub(r"(\([a-z][a-z]\))", "", ph_decoded)
```

Comment:
Phonemizer uses the more general expression `r'\(.+?\)'` from [here](https://github.com/bootphon/phonemizer/blob/fd39cdc0040a8d23646686cdb41cf0144c92e2fd/phonemizer/backend/espeak/language_switch.py#L84) . I think yours could overlook some language with 3-letters code or codes with hyphens. Here is the full language [list](https://github.com/espeak-ng/espeak-ng/blob/master/docs/languages.md).